From 44d54c15bbd8b27b141c15f3062293f4fa86e222 Mon Sep 17 00:00:00 2001
From: Pratham Malaviya <prathamm@mechasystems.com>
Date: Wed, 6 Aug 2025 18:24:37 +0530
Subject: [PATCH] Removed the pins using the sai pad

This patch removes the fec pinctrl which are using the
SAI-1 pad in pinmux.

Signed-off-by: Pratham Malaviya <prathamm@mechasystems.com>

diff --git a/arch/arm/dts/imx8mp-evk.dts b/arch/arm/dts/imx8mp-evk.dts
index 36b860125e0..dcf80d16fd7 100644
--- a/arch/arm/dts/imx8mp-evk.dts
+++ b/arch/arm/dts/imx8mp-evk.dts
@@ -91,26 +91,26 @@
 		status = "okay";
 	};
 
-	rm67199_panel {
-		compatible = "raydium,rm67199";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_mipi_dsi_en>;
-		reset-gpio = <&gpio1 8 GPIO_ACTIVE_LOW>;
-		dsi-lanes = <4>;
-		video-mode = <2>;	/* 0: burst mode
-					 * 1: non-burst mode with sync event
-					 * 2: non-burst mode with sync pulse
-					 */
-		panel-width-mm = <68>;
-		panel-height-mm = <121>;
-		status = "okay";
-
-		port {
-			rm67191_from_dsim: endpoint {
-				remote-endpoint = <&dsim_to_rm67191>;
-			};
-		};
-	};
+	//rm67199_panel {
+	//	compatible = "raydium,rm67199";
+	//	pinctrl-names = "default";
+	//	pinctrl-0 = <&pinctrl_mipi_dsi_en>;
+	//	reset-gpio = <&gpio1 8 GPIO_ACTIVE_LOW>;
+	//	dsi-lanes = <4>;
+	//	video-mode = <2>;	/* 0: burst mode
+	//				 * 1: non-burst mode with sync event
+	//				 * 2: non-burst mode with sync pulse
+	//				 */
+	//	panel-width-mm = <68>;
+	//	panel-height-mm = <121>;
+	//	status = "okay";
+
+	//	port {
+	//		rm67191_from_dsim: endpoint {
+	//			remote-endpoint = <&dsim_to_rm67191>;
+	//		};
+	//	};
+	//};
 
 	cbtl04gp {
 		compatible = "nxp,cbtl04gp";
@@ -405,19 +405,19 @@
 	sda-gpios = <&gpio5 17 GPIO_ACTIVE_HIGH>;
 	status = "okay";
 
-	adv_bridge: adv7535@3d {
-		compatible = "adi,adv7535";
-		reg = <0x3d>;
-		adi,addr-cec = <0x3c>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_from_dsim: endpoint {
-				remote-endpoint = <&dsim_to_adv7535>;
-			};
-		};
-	};
+//	adv_bridge: adv7535@3d {
+//		compatible = "adi,adv7535";
+//		reg = <0x3d>;
+//		adi,addr-cec = <0x3c>;
+//		adi,dsi-lanes = <4>;
+//		status = "disabled";
+//
+//		port {
+//			adv7535_from_dsim: endpoint {
+//				remote-endpoint = <&dsim_to_adv7535>;
+//			};
+//		};
+//	};
 
 	ptn5110: tcpc@50 {
 		compatible = "nxp,ptn5110";
@@ -521,19 +521,19 @@
 &mipi_dsi {
 	status = "okay";
 
-	ports {
-		port@1 {
-			dsim_to_adv7535: endpoint {
-				remote-endpoint = <&adv7535_from_dsim>;
-			};
-		};
-
-		port@2 {
-			dsim_to_rm67191: endpoint {
-				remote-endpoint = <&rm67191_from_dsim>;
-			};
-		};
-	};
+//	ports {
+//		port@1 {
+//			dsim_to_adv7535: endpoint {
+//				remote-endpoint = <&adv7535_from_dsim>;
+//			};
+//		};
+//
+//		port@2 {
+//			dsim_to_rm67191: endpoint {
+//				remote-endpoint = <&rm67191_from_dsim>;
+//			};
+//		};
+//	};
 };
 
 &pcie_phy {
@@ -653,41 +653,41 @@
 &iomuxc {
 	pinctrl_eqos: eqosgrp {
 		fsl,pins = <
-			MX8MP_IOMUXC_ENET_MDC__ENET_QOS_MDC				0x2
-			MX8MP_IOMUXC_ENET_MDIO__ENET_QOS_MDIO				0x2
-			MX8MP_IOMUXC_ENET_RD0__ENET_QOS_RGMII_RD0			0x90
-			MX8MP_IOMUXC_ENET_RD1__ENET_QOS_RGMII_RD1			0x90
-			MX8MP_IOMUXC_ENET_RD2__ENET_QOS_RGMII_RD2			0x90
-			MX8MP_IOMUXC_ENET_RD3__ENET_QOS_RGMII_RD3			0x90
-			MX8MP_IOMUXC_ENET_RXC__CCM_ENET_QOS_CLOCK_GENERATE_RX_CLK	0x90
-			MX8MP_IOMUXC_ENET_RX_CTL__ENET_QOS_RGMII_RX_CTL			0x90
-			MX8MP_IOMUXC_ENET_TD0__ENET_QOS_RGMII_TD0			0x16
-			MX8MP_IOMUXC_ENET_TD1__ENET_QOS_RGMII_TD1			0x16
-			MX8MP_IOMUXC_ENET_TD2__ENET_QOS_RGMII_TD2			0x16
-			MX8MP_IOMUXC_ENET_TD3__ENET_QOS_RGMII_TD3			0x16
-			MX8MP_IOMUXC_ENET_TX_CTL__ENET_QOS_RGMII_TX_CTL			0x16
-			MX8MP_IOMUXC_ENET_TXC__CCM_ENET_QOS_CLOCK_GENERATE_TX_CLK	0x16
-			MX8MP_IOMUXC_SAI2_RXC__GPIO4_IO22				0x10
+	//		MX8MP_IOMUXC_ENET_MDC__ENET_QOS_MDC				0x2
+	//		MX8MP_IOMUXC_ENET_MDIO__ENET_QOS_MDIO				0x2
+	//		MX8MP_IOMUXC_ENET_RD0__ENET_QOS_RGMII_RD0			0x90
+	//		MX8MP_IOMUXC_ENET_RD1__ENET_QOS_RGMII_RD1			0x90
+	//		MX8MP_IOMUXC_ENET_RD2__ENET_QOS_RGMII_RD2			0x90
+	//		MX8MP_IOMUXC_ENET_RD3__ENET_QOS_RGMII_RD3			0x90
+	//		MX8MP_IOMUXC_ENET_RXC__CCM_ENET_QOS_CLOCK_GENERATE_RX_CLK	0x90
+	//		MX8MP_IOMUXC_ENET_RX_CTL__ENET_QOS_RGMII_RX_CTL			0x90
+	//		MX8MP_IOMUXC_ENET_TD0__ENET_QOS_RGMII_TD0			0x16
+	//		MX8MP_IOMUXC_ENET_TD1__ENET_QOS_RGMII_TD1			0x16
+	//		MX8MP_IOMUXC_ENET_TD2__ENET_QOS_RGMII_TD2			0x16
+	//		MX8MP_IOMUXC_ENET_TD3__ENET_QOS_RGMII_TD3			0x16
+	//		MX8MP_IOMUXC_ENET_TX_CTL__ENET_QOS_RGMII_TX_CTL			0x16
+	//		MX8MP_IOMUXC_ENET_TXC__CCM_ENET_QOS_CLOCK_GENERATE_TX_CLK	0x16
+	//		MX8MP_IOMUXC_SAI2_RXC__GPIO4_IO22				0x10
 		>;
 	};
 
 	pinctrl_fec: fecgrp {
 		fsl,pins = <
-			MX8MP_IOMUXC_SAI1_RXD2__ENET1_MDC		0x2
-			MX8MP_IOMUXC_SAI1_RXD3__ENET1_MDIO		0x2
-			MX8MP_IOMUXC_SAI1_RXD4__ENET1_RGMII_RD0		0x90
-			MX8MP_IOMUXC_SAI1_RXD5__ENET1_RGMII_RD1		0x90
-			MX8MP_IOMUXC_SAI1_RXD6__ENET1_RGMII_RD2		0x90
-			MX8MP_IOMUXC_SAI1_RXD7__ENET1_RGMII_RD3		0x90
-			MX8MP_IOMUXC_SAI1_TXC__ENET1_RGMII_RXC		0x90
-			MX8MP_IOMUXC_SAI1_TXFS__ENET1_RGMII_RX_CTL	0x90
-			MX8MP_IOMUXC_SAI1_TXD0__ENET1_RGMII_TD0		0x16
-			MX8MP_IOMUXC_SAI1_TXD1__ENET1_RGMII_TD1		0x16
-			MX8MP_IOMUXC_SAI1_TXD2__ENET1_RGMII_TD2		0x16
-			MX8MP_IOMUXC_SAI1_TXD3__ENET1_RGMII_TD3		0x16
-			MX8MP_IOMUXC_SAI1_TXD4__ENET1_RGMII_TX_CTL	0x16
-			MX8MP_IOMUXC_SAI1_TXD5__ENET1_RGMII_TXC		0x16
-			MX8MP_IOMUXC_SAI1_RXD0__GPIO4_IO02		0x10
+//			MX8MP_IOMUXC_SAI1_RXD2__ENET1_MDC		0x2
+//			MX8MP_IOMUXC_SAI1_RXD3__ENET1_MDIO		0x2
+//			MX8MP_IOMUXC_SAI1_RXD4__ENET1_RGMII_RD0		0x90
+//			MX8MP_IOMUXC_SAI1_RXD5__ENET1_RGMII_RD1		0x90
+//			MX8MP_IOMUXC_SAI1_RXD6__ENET1_RGMII_RD2		0x90
+//			MX8MP_IOMUXC_SAI1_RXD7__ENET1_RGMII_RD3		0x90
+//			MX8MP_IOMUXC_SAI1_TXC__ENET1_RGMII_RXC		0x90
+//			MX8MP_IOMUXC_SAI1_TXFS__ENET1_RGMII_RX_CTL	0x90
+//			MX8MP_IOMUXC_SAI1_TXD0__ENET1_RGMII_TD0		0x16
+//			MX8MP_IOMUXC_SAI1_TXD1__ENET1_RGMII_TD1		0x16
+//			MX8MP_IOMUXC_SAI1_TXD2__ENET1_RGMII_TD2		0x16
+//			MX8MP_IOMUXC_SAI1_TXD3__ENET1_RGMII_TD3		0x16
+//			MX8MP_IOMUXC_SAI1_TXD4__ENET1_RGMII_TX_CTL	0x16
+//			MX8MP_IOMUXC_SAI1_TXD5__ENET1_RGMII_TXC		0x16
+//			MX8MP_IOMUXC_SAI1_RXD0__GPIO4_IO02		0x10
 		>;
 	};
 
-- 
2.25.1

